app/views/mailers/mailshots_mailer/company_support_testimonial.html.erb

```diff
@@ -0,0 +1,606 @@
+          <div class="layout__inner" style="border-collapse: collapse;display: table;width: 100%;background-color: #ffffff;">
+          <!--[if (mso)|(IE)]><table align="center" cellpadding="0" cellspacing="0" role="presentation"><tr class="layout-fixed-width" style="background-color: #ffffff;"><td style="width: 600px" class="w560"><![endif]-->
+            <div class="column" style="text-align: left;color: #130B43;font-size: 16px;line-height: 25px;font-family: Poppins,Arial,'Helvetica Neue',Helvetica,sans-serif;">
+
+              <div style="margin-left: 20px;margin-right: 20px;">
+                <div style="mso-line-height-rule: exactly;mso-text-raise: 11px;vertical-align: middle;">
+                  <p style="margin-top: 16px;margin-bottom: 0;">
+                    Hi <%= @user.handle %>,
+                  </p>
+                  <p style="margin-top: 20px;margin-bottom: 0;">
+                  I hope you've found Exercism a helpful way of improving your programming skills and that our unique blend of teaching, practice and mentoring has been fun!
+                  </p>
+                  <p style="margin-top: 20px;margin-bottom: 0;">
+                    Now we need your help in keeping the lights on at Exercism for another year! We're hoping you might be able to <strong style="font-weight:600">get your company to support us</strong>. A $1k donation is often a small amount for a company but a huge amount for us. Could you introduce us to someone at your company who could support Exercism?
+                  </p>
+                  <p style="margin-top: 20px;margin-bottom: 0;">
+                    Some people are also able to support Exercism through their corporate learning or education allowance. They're allowed to spend up to a certain amount on their professional development and choose to donate some of it to Exercism. Could you do the same?
+                  </p>
+                  <p style="margin-top: 20px;margin-bottom: 0;">
+                    As a reminder, Exercism is a not-for-profit organisation. We provided over 100 million minutes of free coding education and hundreds of thousands of mentoring sessions. We rely entirely on donations from individuals and companies to survive. You can learn more about our impact on our <a href="https://exercism.org/about/impact" style="font-weight:600; color:#2E57E8; text-decoration: underline">impact page</a>.
+                  </p>
+                  <p style="margin-top: 20px;margin-bottom: 0;">
+                    If you'd like more information on either idea, please let me know!                    
+                  </p>
+                  <p style="margin-top: 20px;margin-bottom: 20px;">
+                    <p style="margin-top: 0;margin-bottom: 0;">Thanks,<br/>
+                    Loretta Bresciani Murray<br/>
+                    Head of Partnerships and Fundraising of Exercism</p>
```

Comment:
I've made a small tweak.
The original email template has this as:

> Thanks,
> Loretta

The other mailshot (community_launch) though used:

> Kind regards,
> 
> Jeremy Walker,
> CEO of Exercism

I've applied the latter's structure to this email. Let me know if that is correct.